bigquery_datatransfer/tests/system/gapic/v1/test_system_data_transfer_service_v1.py

```diff
@@ -26,5 +26,5 @@ def test_list_data_sources(self):
         project_id = os.environ["PROJECT_ID"]
 
         client = bigquery_datatransfer_v1.DataTransferServiceClient()
-        parent = client.project_path(project_id)
+        parent = client.location_path(project_id, "us-central1")
```

Comment:
Oh, we have generated system tests, now? I don't know why this changed, but `location_path` is broken according to Kokoro.